### PR TITLE
docs: Update docs for v1.2.0

### PR DESCRIPTION
**Pull request by LeeDongGeon1996**
### Description

- general docs update.

### Checklist

- [x] My code follows the existing style, code structure, and naming taxonomy
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my own code and included any verifying manual calculations
- [x] I have added or updated unit tests that prove my fix is effective or that my feature works, and achieves sufficient code coverage.  New and existing unit tests pass locally and in the build (below) with my changes
- [x] My changes generate no new warnings and running code analysis does not produce any issues
- [x] I have added or run the performance tests that depict optimal execution times
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 [![PyPI](https://img.shields.io/pypi/v/stock-indicators?color=blue&label=PyPI)](https://badge.fury.io/py/stock-indicators)
 [![code coverage](https://img.shields.io/azure-devops/coverage/skender/stock.indicators/26/main?logo=AzureDevOps&label=Test%20Coverage)](https://dev.azure.com/skender/Stock.Indicators/_build/latest?definitionId=26&branchName=main&view=codecoverage-tab)
```
Comment (DaveSkender) [inline README.md:4]: Not needed for this PR, but after we've figured out how to get the [coverage badge from Codacy](https://github.com/DaveSkender/Stock.Indicators.Python/pull/336), we'll need to change this from the obsolete Azure DevOps source.

**Review by DaveSkender — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (LeeDongGeon1996) [reply README.md:4]: Right. We should replace. Are you having a problem with that task? Or just doing now? (Just a question😅)

Comment (DaveSkender) [reply README.md:4]: I started it, but for some reason the badge didn't work, so I stopped working on it.  Feel free to try it out.  Maybe you'll have better luck.  No need to hold up this PR though.

Comment (LeeDongGeon1996) [reply README.md:4]: Okay, Please add me Codacy project and TestPypi.

**Pull request merged**